tests/app/main/views/test_index.py

```diff
@@ -105,26 +115,50 @@ def test_hiding_pages_from_search_engines(
 )
 def test_static_pages(client_request, mock_get_organization_by_domain, view, mocker):
     mocker.patch("app.notify_client.user_api_client.UserApiClient.deactivate_user")
-    request = partial(client_request.get, "main.{}".format(view))
 
-    # Check the page loads when user is signed in
-    page = request()
-    assert page.select_one("meta[name=description]")
+    # Function to check if a view is feature-flagged and should return 404 when disabled
+    def is_feature_flagged(view):
 
-    # Check it still works when they don’t have a recent service
-    with client_request.session_transaction() as session:
-        session["service_id"] = None
-    request()
+        feature_best_practices_enabled = (
+            os.getenv("FEATURE_BEST_PRACTICES_ENABLED", "false").lower() == "true"
+        )
+        feature_flagged_views = [
+            "best_practices",
+            "clear_goals",
+            "rules_and_regulations",
+            "establish_trust",
+            "write_for_action",
+            "multiple_languages",
+            "benchmark_performance",
+        ]
+        return not feature_best_practices_enabled and view in feature_flagged_views
```

Comment:
This _should_ work like this:

```suggestion
        return not current_app.config["FEATURE_BEST_PRACTICES_ENABLED"] and view in feature_flagged_views
```